spinalcordtoolbox/reports/qc.py

```diff
@@ -28,6 +28,8 @@
 from spinalcordtoolbox.image import Image
 import spinalcordtoolbox.reports.slice as qcslice
 
+import portalocker
```

Comment:
This 3rd party module should be imported before the modules from this library and after importing built-in modules.